examples/multilang-multi-project/proto/src/api/v1/api-audio.proto

```
@@ -1,11 +1,10 @@
 syntax = "proto3";
 package audio.v1;
 
+import "buf/validate/validate.proto";
```

Buf (1.54.0)

import "buf/validate/validate.proto": file does not exist

(COMPILE)